examples/demo_mult_image.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+# title           : this_python_file.py
+# description     :This will create a header for a python script.
```

Comment:
This example seems redundant with the `add_images` example which adds two images, and also doesn't contain anything specific to the vectors. Can you remove from this PR

Reply:
removed